speedcord/http.py

```diff
@@ -26,11 +34,19 @@ def __init__(self, method, route, **parameters):
 
     @property
     def bucket(self):
+        """
+        The Route's bucket identifier.
+        """
         return f"{self.channel_id}:{self.guild_id}:{self.path}"
 
 
 class LockManager:
     def __init__(self, lock: asyncio.Lock):
+        """
+        Used by HttpClient to handle rate limits. Locked when a Bucket's rate limit has been
```

Comment:
Maybe change this to something like

Adds a way to stop the lock from unlocking to the normal asyncio.Lock context manager